MANIFEST.in

```diff
@@ -1,2 +1,7 @@
+graft .vscode
 graft demo
 graft jupyterhub_moss
+include .bumpversion.cfg
+include .editorconfig
+include .prettierrc.json
+include CONTRIBUTING.md
```

Comment:
I think all this stuff is related to the development and should not be included in the package.

See [h5grove's manifest](https://github.com/silx-kit/h5grove/blob/main/MANIFEST.in) for example

Reply:
Done!